MANIFEST.in

```diff
@@ -1,4 +1,4 @@
-include README.rst CHANGES.rst
+include README.rst CHANGES.rst requirements.txt requirements-windows.txt
```

Comment:
Any reason about adding `requirements-windows.txt`, 
the `requirements-windows.txt` is not used in the `setup.py`,
he is only used in the `Makefile`,
and he just add directly a wheel, then for me we don't need it, and he hasn't anything useful in the egg...

Reply:
OK thanks, will remove that